test/command/suite/load/time/cast/string/iso8601/valid.test

```diff
@@ -0,0 +1,26 @@
+plugin_register functions/time
+table_create Logs TABLE_NO_KEY
+column_create Logs created_at COLUMN_SCALAR Time
+column_create Logs created_at_text COLUMN_SCALAR ShortText
+
+
+
+load --table Logs
+[
+{"created_at": "2000-01-01T10:00:00Z","created_at_text": "2000-01-01T19:00:00.000000+09:00"},
```

Comment:
Could you specify the same text as the `created_at` to `created_at_text` instead of expected value to view input in the `.expected` file?

(And a space after `,` is missing.)

It may be better that we also have a label what is tested by this data:

```json
{"_key": "timezone: Z", "created_at": "2000-01-01T10:00:00Z", "created_at_text": "2000-01-01T10:00:00Z"}
```

Reply:
Ah, I understand now. I have fixed it.